src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/Breadcrumb.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.ui.stepsdesign;
+
+import io.cdap.cdap.ui.utils.Helper;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.WaitHelper;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import io.cucumber.java.en.When;
+
+public class Breadcrumb {
+    ChromeDriver driver;
+
+    @Given("Navigate to the home page")
+    public void navigateToTheHomePage() {
+        driver = new ChromeDriver();
```

Comment:
why don't you do `import io.cdap.e2e.utils.SeleniumDriver;` and use `SeleniumDriver.openPage(url)` instead?
you can also do `WebDriver driver = SeleniumDriver.getDriver();`  and use `driver` when needed.

Reply:
we generally rely on generic methods defined in the e2e-tests source repo. For example for clicking on elements, we use:
`ElementHelper.clickOnElement(Helper.locateElementByTestId(<datatestId>));`

Reply:
Done.